Index: app/src/main/java/edu/up/cs301/pig/PigHumanPlayer.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/edu/up/cs301/pig/PigHumanPlayer.java b/app/src/main/java/edu/up/cs301/Yahtzee/YahtzeeHumanPlayer.java
rename from app/src/main/java/edu/up/cs301/pig/PigHumanPlayer.java
rename to app/src/main/java/edu/up/cs301/Yahtzee/YahtzeeHumanPlayer.java
--- a/app/src/main/java/edu/up/cs301/pig/PigHumanPlayer.java	(revision 7d7d4074994a3bef42c6cc9d2ab9b26db8bc07aa)
+++ b/app/src/main/java/edu/up/cs301/Yahtzee/YahtzeeHumanPlayer.java	(date 1647535967003)
@@ -1,12 +1,10 @@
-package edu.up.cs301.pig;
+package edu.up.cs301.Yahtzee;
 
 import edu.up.cs301.game.GameHumanPlayer;
 import edu.up.cs301.game.GameMainActivity;
 import edu.up.cs301.game.R;
 import edu.up.cs301.game.infoMsg.GameInfo;
 
-import android.graphics.Color;
-import android.graphics.drawable.Drawable;
 import android.view.View;
 import android.widget.Button;
 import android.widget.ImageButton;
@@ -20,7 +18,7 @@
  * @author Andrew M. Nuxoll, modified by Steven R. Vegdahl
  * @version February 2016
  */
-public class PigHumanPlayer extends GameHumanPlayer implements OnClickListener {
+public class YahtzeeHumanPlayer extends GameHumanPlayer implements OnClickListener {
 
 	/* instance variables */
 
@@ -38,7 +36,7 @@
     /**
      * constructor does nothing extra
      */
-    public PigHumanPlayer(String name) {
+    public YahtzeeHumanPlayer(String name) {
         super(name);
     }
 
Index: app/src/main/java/edu/up/cs301/pig/YahtzeeScore.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/edu/up/cs301/pig/YahtzeeScore.java b/app/src/main/java/edu/up/cs301/Yahtzee/YahtzeeScore.java
rename from app/src/main/java/edu/up/cs301/pig/YahtzeeScore.java
rename to app/src/main/java/edu/up/cs301/Yahtzee/YahtzeeScore.java
--- a/app/src/main/java/edu/up/cs301/pig/YahtzeeScore.java	(revision 7d7d4074994a3bef42c6cc9d2ab9b26db8bc07aa)
+++ b/app/src/main/java/edu/up/cs301/Yahtzee/YahtzeeScore.java	(date 1647535967044)
@@ -1,4 +1,4 @@
-package edu.up.cs301.pig;
+package edu.up.cs301.Yahtzee;
 
 import edu.up.cs301.game.GamePlayer;
 import edu.up.cs301.game.actionMsg.GameAction;
Index: app/src/main/java/edu/up/cs301/pig/YahtzeeRoll.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/edu/up/cs301/pig/YahtzeeRoll.java b/app/src/main/java/edu/up/cs301/Yahtzee/YahtzeeRoll.java
rename from app/src/main/java/edu/up/cs301/pig/YahtzeeRoll.java
rename to app/src/main/java/edu/up/cs301/Yahtzee/YahtzeeRoll.java
--- a/app/src/main/java/edu/up/cs301/pig/YahtzeeRoll.java	(revision 7d7d4074994a3bef42c6cc9d2ab9b26db8bc07aa)
+++ b/app/src/main/java/edu/up/cs301/Yahtzee/YahtzeeRoll.java	(date 1647535967028)
@@ -1,4 +1,4 @@
-package edu.up.cs301.pig;
+package edu.up.cs301.Yahtzee;
 
 import edu.up.cs301.game.GamePlayer;
 import edu.up.cs301.game.actionMsg.GameAction;
Index: app/src/main/java/edu/up/cs301/pig/PigComputerPlayer.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/edu/up/cs301/pig/PigComputerPlayer.java b/app/src/main/java/edu/up/cs301/Yahtzee/YahtzeeComputerPlayer.java
rename from app/src/main/java/edu/up/cs301/pig/PigComputerPlayer.java
rename to app/src/main/java/edu/up/cs301/Yahtzee/YahtzeeComputerPlayer.java
--- a/app/src/main/java/edu/up/cs301/pig/PigComputerPlayer.java	(revision 7d7d4074994a3bef42c6cc9d2ab9b26db8bc07aa)
+++ b/app/src/main/java/edu/up/cs301/Yahtzee/YahtzeeComputerPlayer.java	(date 1647535966995)
@@ -1,9 +1,7 @@
-package edu.up.cs301.pig;
+package edu.up.cs301.Yahtzee;
 
 import edu.up.cs301.game.GameComputerPlayer;
-import edu.up.cs301.game.actionMsg.GameAction;
 import edu.up.cs301.game.infoMsg.GameInfo;
-import edu.up.cs301.game.util.Tickable;
 
 /**
  * An AI for Pig
@@ -11,12 +9,12 @@
  * @author Andrew M. Nuxoll
  * @version August 2015
  */
-public class PigComputerPlayer extends GameComputerPlayer {
+public class YahtzeeComputerPlayer extends GameComputerPlayer {
 
     /**
      * ctor does nothing extra
      */
-    public PigComputerPlayer(String name) {
+    public YahtzeeComputerPlayer(String name) {
         super(name);
     }
 
Index: app/src/main/java/edu/up/cs301/pig/Dice.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/edu/up/cs301/pig/Dice.java b/app/src/main/java/edu/up/cs301/Yahtzee/Dice.java
rename from app/src/main/java/edu/up/cs301/pig/Dice.java
rename to app/src/main/java/edu/up/cs301/Yahtzee/Dice.java
--- a/app/src/main/java/edu/up/cs301/pig/Dice.java	(revision 7d7d4074994a3bef42c6cc9d2ab9b26db8bc07aa)
+++ b/app/src/main/java/edu/up/cs301/Yahtzee/Dice.java	(date 1647535966978)
@@ -1,4 +1,4 @@
-package edu.up.cs301.pig;
+package edu.up.cs301.Yahtzee;
 
 public class Dice {
     private int val;
Index: app/src/main/java/edu/up/cs301/pig/YahtzeeKeep.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/edu/up/cs301/pig/YahtzeeKeep.java b/app/src/main/java/edu/up/cs301/Yahtzee/YahtzeeKeep.java
rename from app/src/main/java/edu/up/cs301/pig/YahtzeeKeep.java
rename to app/src/main/java/edu/up/cs301/Yahtzee/YahtzeeKeep.java
--- a/app/src/main/java/edu/up/cs301/pig/YahtzeeKeep.java	(revision 7d7d4074994a3bef42c6cc9d2ab9b26db8bc07aa)
+++ b/app/src/main/java/edu/up/cs301/Yahtzee/YahtzeeKeep.java	(date 1647535967020)
@@ -1,4 +1,4 @@
-package edu.up.cs301.pig;
+package edu.up.cs301.Yahtzee;
 
 import edu.up.cs301.game.GamePlayer;
 import edu.up.cs301.game.actionMsg.GameAction;
Index: app/src/main/java/edu/up/cs301/pig/YahtzeeGameState.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/edu/up/cs301/pig/YahtzeeGameState.java b/app/src/main/java/edu/up/cs301/Yahtzee/YahtzeeGameState.java
rename from app/src/main/java/edu/up/cs301/pig/YahtzeeGameState.java
rename to app/src/main/java/edu/up/cs301/Yahtzee/YahtzeeGameState.java
--- a/app/src/main/java/edu/up/cs301/pig/YahtzeeGameState.java	(revision 7d7d4074994a3bef42c6cc9d2ab9b26db8bc07aa)
+++ b/app/src/main/java/edu/up/cs301/Yahtzee/YahtzeeGameState.java	(date 1647535967053)
@@ -1,4 +1,4 @@
-package edu.up.cs301.pig;
+package edu.up.cs301.Yahtzee;
 
 import edu.up.cs301.game.infoMsg.GameState;
 
Index: app/src/main/java/edu/up/cs301/pig/YahtzeeLocalGame.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/edu/up/cs301/pig/YahtzeeLocalGame.java b/app/src/main/java/edu/up/cs301/Yahtzee/YahtzeeLocalGame.java
rename from app/src/main/java/edu/up/cs301/pig/YahtzeeLocalGame.java
rename to app/src/main/java/edu/up/cs301/Yahtzee/YahtzeeLocalGame.java
--- a/app/src/main/java/edu/up/cs301/pig/YahtzeeLocalGame.java	(revision 7d7d4074994a3bef42c6cc9d2ab9b26db8bc07aa)
+++ b/app/src/main/java/edu/up/cs301/Yahtzee/YahtzeeLocalGame.java	(date 1647540022640)
@@ -1,4 +1,4 @@
-package edu.up.cs301.pig;
+package edu.up.cs301.Yahtzee;
 
 import edu.up.cs301.game.GamePlayer;
 import edu.up.cs301.game.LocalGame;
@@ -82,7 +82,7 @@
     @Override
     protected void sendUpdatedStateTo(GamePlayer p) {
         //TODO  You will implement this method
-        PigGameState updatedGameState = new PigGameState(masterGameState);
+        YahtzeeGameState updatedGameState = new YahtzeeGameState(masterGameState);
         p.sendInfo(updatedGameState);
     }//sendUpdatedSate
 
Index: app/src/main/java/edu/up/cs301/pig/PigMainActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/edu/up/cs301/pig/PigMainActivity.java b/app/src/main/java/edu/up/cs301/Yahtzee/YahtzeeMainActivity.java
rename from app/src/main/java/edu/up/cs301/pig/PigMainActivity.java
rename to app/src/main/java/edu/up/cs301/Yahtzee/YahtzeeMainActivity.java
--- a/app/src/main/java/edu/up/cs301/pig/PigMainActivity.java	(revision 7d7d4074994a3bef42c6cc9d2ab9b26db8bc07aa)
+++ b/app/src/main/java/edu/up/cs301/Yahtzee/YahtzeeMainActivity.java	(date 1647876786245)
@@ -1,4 +1,4 @@
-package edu.up.cs301.pig;
+package edu.up.cs301.Yahtzee;
 
 import java.util.ArrayList;
 
@@ -8,13 +8,14 @@
 import edu.up.cs301.game.config.GameConfig;
 import edu.up.cs301.game.config.GamePlayerType;
 
+
 /**
  * this is the primary activity for Pig
  *
  * @author Andrew M. Nuxoll, modified by Steven R. Vegdahl
  * @version February 2016
  */
-public class PigMainActivity extends GameMainActivity {
+public class YahtzeeMainActivity extends GameMainActivity {
 
     // the port number that this game will use when playing over the network
     private static final int PORT_NUMBER = 2278;
@@ -36,11 +37,11 @@
         // Pig has two player types:  human and computer
         playerTypes.add(new GamePlayerType("Local Human Player") {
             public GamePlayer createPlayer(String name) {
-                return new PigHumanPlayer(name);
+                return new YahtzeeHumanPlayer(name);
             }});
         playerTypes.add(new GamePlayerType("Computer Player") {
             public GamePlayer createPlayer(String name) {
-                return new PigComputerPlayer(name);
+                return new YahtzeeComputerPlayer(name);
             }});
 
         // Create a game configuration class for Pig:
@@ -60,7 +61,7 @@
      */
     @Override
     public LocalGame createLocalGame() {
-        return new PigLocalGame();
+        return new YahtzeeMainActivity();
     }
 
 }
Index: app/src/main/AndroidManifest.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><manifest xmlns:android=\"http://schemas.android.com/apk/res/android\"\r\n    package=\"edu.up.cs301.game\"\r\n    android:versionCode=\"1\"\r\n    android:versionName=\"1.0\" >\r\n\r\n    <uses-permission android:name=\"android.permission.INTERNET\" />\r\n    <uses-permission android:name=\"android.permission.ACCESS_NETWORK_STATE\" />\r\n\r\n    <application\r\n        android:icon=\"@drawable/face1\"\r\n        android:label=\"@string/app_name\"\r\n        android:theme=\"@style/AppTheme\" >\r\n        \r\n        <activity android:name=\"edu.up.cs301.pig.PigMainActivity\"\r\n            android:configChanges=\"keyboard|keyboardHidden|orientation|screenSize\" >\r\n            <intent-filter>\r\n                <action android:name=\"android.intent.action.MAIN\" />\r\n                <category android:name=\"android.intent.category.LAUNCHER\" />\r\n            </intent-filter>\r\n        </activity>\r\n        \r\n    </application>\r\n\r\n</manifest>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/AndroidManifest.xml b/app/src/main/AndroidManifest.xml
--- a/app/src/main/AndroidManifest.xml	(revision 7d7d4074994a3bef42c6cc9d2ab9b26db8bc07aa)
+++ b/app/src/main/AndroidManifest.xml	(date 1647535966987)
@@ -11,7 +11,7 @@
         android:label="@string/app_name"
         android:theme="@style/AppTheme" >
         
-        <activity android:name="edu.up.cs301.pig.PigMainActivity"
+        <activity android:name="edu.up.cs301.Yahtzee.YahtzeeMainActivity"
             android:configChanges="keyboard|keyboardHidden|orientation|screenSize" >
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
